ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -612,6 +612,39 @@ tests:
       ASSISTED_ZTP_CONFIG: |
         EXTRA_HYPERSHIFT_INSTALL_FLAGS="--enable-defaulting-webhook true"
     workflow: assisted-baremetal-operator-capi
+- as: e2e-ai-operator-disconnected-capi
```

Comment:
```suggestion
- as: e2e-ai-operator-disconnected-capi
  always_run: false
```
we can put this option here now instead of updating manually the generated job

Reply:
Gotcha, thanks! I've modified it as suggested - does the other job need this too?

Reply:
nope, it's only for the presubmits

Reply:
Thanks!